test/libs/data-structures/IncrementalMerkleTree.js

```diff
@@ -0,0 +1,145 @@
+const { assert } = require("chai");
+
```

Comment:
```suggestion
```

Reply:
I think it's better to visually distinguish between different import groups (from node, local files, etc.) 
What do you think?

Reply:
Maybe, but let's stick to the already existing tests